lib/mistral_ai_llm_controller_example/prompt.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:mistral_ai_chat_example_app/mistral_ai_llm_controller_example/model.dart';
+
+String controllerDescription(ControllerSettings currentSettings) =>
+    'You are a helpful assistant designed to output only JSON. '
+    'You are given command and a set of functions that you can perform. '
+    'Based on the command you should chose one of the provided functions '
+    'If you do not understand the command, you should return a following JSON. '
+    '{ "name": "unknown", "parameters": "" } '
+    'You can perform one function at a time. '
+    'Do not suggest new functions. '
+    'Do not return anything else than JSON. '
+    'Do not explain anything. '
+    'Current settings: '
+    '{ "color": ${currentSettings.color}, '
+    '"temperature": ${currentSettings.temperature}, '
+    '"volume": ${currentSettings.volume}}';
+
```

Comment:
So this is the propmt as a context to AI model?

Reply:
Updated in [f1d0422](https://github.com/nomtek/flutter_ai_examples/pull/6/commits/f1d04224d52e040b4c0ba8e59970fa25afc5a5d3), [0b46f3d](https://github.com/nomtek/flutter_ai_examples/pull/6/commits/0b46f3d98e00d64f9aae3b960805c6f38b54ba64)